### PR TITLE
Design

### PR DESCRIPTION
**Pull request by annaiandresen**
- Add tailwind css config 
- Change sidebar design
- Temporarily remove punishment grid

This PR is mainly here to implement tailwind css in master. 

!Fetching groups does not work!

### CODE REVIEW

File: frontend/src/Components/Left_navbar_components/Accordion.svelte [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <script lang="ts">
   export let title: string;
-  export let color: string = "#0d5474";
+  export let color: string = "#004372";
```
Comment (jfremstad) [inline frontend/src/Components/Left_navbar_components/Accordion.svelte:3]: Maybe we could use one of the colors defined in tailwindcss?

File: frontend/src/Components/PunishmentDisplay/PunishmentGrid.svelte [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      margin-left: auto;
+      margin-right: auto;
+      text-align: center;
+  }
+
+  .name-col{
+    background-color: green;
+  }
+
+  .punishmens-col {
```
Comment (jfremstad) [inline frontend/src/Components/PunishmentDisplay/PunishmentGrid.svelte:89]: .punishments-col?

File: frontend/src/Components/Top_group_picker_components/GroupLogos.svelte [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <script lang="ts">
   import GroupStore from "../../stores/groups";
   import GroupButton from "./GroupButton.svelte";
+  //import { getGroups } from "../../api";
```
Comment (jfremstad) [inline frontend/src/Components/Top_group_picker_components/GroupLogos.svelte:4]: Remove this line?

**Review by jfremstad — APPROVED** [review]

File: frontend/src/Components/Left_navbar_components/Accordion.svelte (hunk 1 above)
Comment (annaiandresen) [reply frontend/src/Components/Left_navbar_components/Accordion.svelte:3]: Can have a look at that later 8)

**Pull request merged**